src/module/actor/sheets/SR5VehicleActorSheet.ts

```diff
@@ -123,11 +111,54 @@ export class SR5VehicleActorSheet extends SR5MatrixActorSheet {
         };
     }
 
+    static override TABS = {
+        ...super.TABS,
+        primary: {
+            initial: 'skills',
+            labelPrefix: 'SR5.Tabs.Actor',
+            tabs: [
+                { id: 'actions', label: 'SR5.Tabs.Actor.Actions', cssClass: '' },
+                { id: 'skills', label: 'SR5.Tabs.Actor.Vehicle', cssClass: '' },
+                { id: 'inventory', label: 'SR5.Tabs.Actor.Inventory', cssClass: '' },
+                { id: 'matrix', label: 'SR5.Tabs.Actor.Matrix', cssClass: '' },
+                { id: 'effects', label: 'SR5.Tabs.Actor.Effects', cssClass: '' },
+                { id: 'description', label: '', icon: 'far fa-info', tooltip: 'SR5.Tooltips.Actor.Description', cssClass: 'skinny' },
+                { id: 'misc', label: '', icon: 'fas fa-gear', tooltip: 'SR5.Tooltips.Actor.MiscConfig', cssClass: 'skinny' },
+            ]
+        },
+    }
+
+    static override PARTS = {
+        ...super.PARTS,
+        matrix: {
+            template: SheetFlow.templateBase('actor/tabs/matrix'),
+            scrollable: [
+                '#matrix-actions-scroll',
+                '#marked-icons-scroll' ,
+                '#owned-icons-scroll',
+                '#network-icons-scroll',
+                '#programs-scroll',
+            ]
+        },
+        skills: {
+            template: SheetFlow.templateBase('actor/tabs/vehicle-skills'),
+            templates: [...SheetFlow.templateActorSystemParts(
+                'active-skills', 'vehicle-attributes'
+            ), ...SheetFlow.templateListItem('skill')],
+            scrollable: ['#active-skills-scroll']
+        },
+        inventory: {
+            template: SheetFlow.templateBase('actor/tabs/inventory'),
+            scrollable: ['.scrollable']
+        },
+    }
+
+
     /**
      * Connect to the PAN of the Driver
      * @param event
      */
-    async _onConnectToDriver(event) {
+    static async #connectToDriver(this: SR5VehicleActorSheet, event) {
```

Comment:
nice trick to make a static function to think it's not static. Since it's marked as private, it shouldn't be a problem

Reply:
This appears to be the "standard" thing to do on sheets, found from a few other foundry systems. TBH I hate it but it works and is "the foundry standard" so oh well